project/client/js/google-auth.js

```diff
@@ -0,0 +1,42 @@
+var googleUser = {};
+var initGoogleAuth = function() {
+    gapi.load('auth2', function(){
+        // Retrieve the singleton for the GoogleAuth library and set up the client.
+        auth2 = gapi.auth2.init({
+            client_id: '467471910617-vo4g6ojobn4o2tu25vo38ofurfis0jqj.apps.googleusercontent.com',
```

Comment:
I guess for now I can just update with one I obtain myself and make a note to tackle this when I get a build pipeline sorted.